net/scripts/azure-provider.sh

```diff
@@ -222,13 +228,12 @@ cloud_CreateInstances() {
       az vm create --name "$nodeName" "${args[@]}" --no-wait
     done
 
-    # 3: Wait until all nodes are created
-    for nodeName in "${nodes[@]}"; do
-      az vm wait --created --name "$nodeName" --resource-group "$networkName"
-    done
-
-    # 4. If GPU is to be enabled, install the appropriate extension
+    # 3. If GPU is to be enabled, wait until nodes are created, then install the appropriate extension
```

Comment:
Will running this in a subshell in background help speed up the network bootup?

Reply:
I'm not sure.  The --no-wait flag in ln#228 causes the cmd to return right away, once the operation is started at the Azure server side, so most of the creation activity all gets started at the same time without blocking the script.  The am vz wait command in ln#234 is gated to check that node N is finished creating only after node N-1 returns.  This usually runs through pretty quick as all the VMs finish getting created around the same time.  Though I have seen that one VM in the middle of the list might not get provisioned correctly and hold up the wait for loop until it hits the timeout.

Reply:
Sounds good 👍